src/compressed_tensors/compressors/quantized_compressors/pack_quantized.py

```diff
@@ -133,14 +159,29 @@ def decompress_weight(
         original_shape = torch.Size(compressed_data["weight_shape"])
         num_bits = quantization_args.num_bits
         unpacked = unpack_from_int32(weight, num_bits, original_shape)
+
+        # NOTE: this will fail decompression as we don't currently handle packed zp on decompression
+        if not quantization_args.symmetric and quantization_args.strategy in [
+            QuantizationStrategy.GROUP.group,
+            QuantizationStrategy.CHANNEL.group,
+        ]:
+            raise ValueError(
+                "Decompression of packed zero points is currently not supported"
+            )
+            assert zero_point is not None
+            original_zp_shape = (original_shape[0], scale.shape[-1])
+            zero_point = unpack_from_int32(
+                zero_point, num_bits, original_zp_shape, packed_dim=0
+            )
+
         decompressed_weight = dequantize(
             x_q=unpacked, scale=scale, zero_point=zero_point, g_idx=g_idx
         )
 
         return decompressed_weight
 
 
-def pack_to_int32(value: torch.Tensor, num_bits: int) -> torch.Tensor:
+def pack_to_int32(value: torch.Tensor, num_bits: int, packed_dim=1) -> torch.Tensor:
```

Comment:
it seems like `packed_dim` should only ever be 0 or 1, right? Curious if we do things like this to help users understand
```suggestion
def pack_to_int32(value: torch.Tensor, num_bits: int, packed_dim:typing.Union[typing.Literal[0], typing.Literal[1]] = 1) -> torch.Tensor:
```